monitor-openqa_job

```diff
@@ -27,17 +27,17 @@ JOB_ID=$(filter_id job_post_response)
 echo "waiting for job ${JOB_ID} to finish"
 while sleep "${sleep_time}" ; do
     # shellcheck disable=SC2086
-    $openqa_client --host "${host}" jobs/${JOB_ID} | grep -q 'state.*done' && break
+    [ "$($openqa_cli api jobs/"${JOB_ID}" | jq -r '.job.state')" = 'done' ] && break
```

Comment:
you removed the host parameter, see http://jenkins.qa.suse.de/job/monitor-openQA_in_openQA-TW/7187/console for the problem